webpack-plugin.js

```diff
@@ -0,0 +1,22 @@
+function myPlugin(
+    context,
+    opts,
+) {
+    return {
+        name: 'webpack-plugin',
```

Comment:
use more description name for this name and for the function name eg 'webpack-custom-config' (if this is still necessary)

Reply:
Done